roqoqo/tests/integration/operations/spin_boson_operations.rs

```diff
@@ -331,6 +435,114 @@ fn jaynes_cummings_serde() {
     );
 }
 
+#[cfg(feature = "serialize")]
+#[test]
+fn single_excitation_load_serde() {
+    let op = SingleExcitationLoad::new(0, 0);
+
+    assert_tokens(
+        &op.clone().readable(),
+        &[
+            Token::Struct {
+                name: "SingleExcitationLoad",
+                len: 2,
+            },
+            Token::Str("qubit"),
+            Token::U64(0),
+            Token::Str("mode"),
+            Token::U64(0),
+            Token::StructEnd,
+        ],
+    );
+
+    assert_tokens(
+        &op.compact(),
+        &[
+            Token::Struct {
+                name: "SingleExcitationLoad",
+                len: 2,
+            },
+            Token::Str("qubit"),
+            Token::U64(0),
+            Token::Str("mode"),
+            Token::U64(0),
+            Token::StructEnd,
+        ],
+    );
+}
+
+#[cfg(feature = "serialize")]
+#[test]
+fn single_excitation_store_serde() {
+    let op = SingleExcitationStore::new(0, 0);
+
+    assert_tokens(
+        &op.clone().readable(),
+        &[
+            Token::Struct {
+                name: "SingleExcitationStore",
+                len: 2,
+            },
+            Token::Str("qubit"),
+            Token::U64(0),
+            Token::Str("mode"),
+            Token::U64(0),
+            Token::StructEnd,
+        ],
+    );
+
+    assert_tokens(
+        &op.compact(),
+        &[
+            Token::Struct {
+                name: "SingleExcitationStore",
+                len: 2,
+            },
+            Token::Str("qubit"),
+            Token::U64(0),
+            Token::Str("mode"),
+            Token::U64(0),
+            Token::StructEnd,
+        ],
+    );
+}
+
+#[cfg(feature = "serialize")]
```

Comment:
These three added tests could arguably be turned into one using `test_case`, but I don't mind either way

Reply:
I've spent some hours trying to do that but the `Token::Struct` wants an entry with a `'static` lifetime, which would complicate the test needlessly. Do you know of an easy way to solve this?